src/actions/weeklySummaries.js

```diff
@@ -79,7 +79,8 @@ export const updateWeeklySummaries = (userId, weeklySummariesData) => {
 
       // Merge the weekly summaries related changes with the user's profile.
       const {mediaUrl, weeklySummaries, weeklySummariesCount } = weeklySummariesData;
-      console.log('respon get', response.data)
+      
+      // console.log('respon get', response.data)
```

Comment:
Remove console.log

Reply:
I removed the console.log from the weeklySummaries.js archive.

![photo 4](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73541319/55818faa-3e96-4c21-9b04-bdeedf6e3432)